templates/Dockerfile-deb.tmpl

```diff
@@ -72,8 +79,7 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main 15'
         liblz4-1 \
         libzstd1 \
         libzstd-dev \
-        sudo \
-    && rm -rf /var/lib/apt/lists/*
```

Comment:
this line should be added back again to avoid docker image bloat

Reply:
Fixed